prow/config/config.go

```diff
@@ -392,7 +393,7 @@ func LoadSingleSecret(path string) ([]byte, error) {
 	if err != nil {
 		return nil, fmt.Errorf("error reading %s: %v", path, err)
 	}
-	return b, nil
+	return bytes.TrimSpace(b), nil
```

Comment:
Doesn't this just mean the secret is incorrect? /shrug

Reply:
Yeah, though this is what we used to do before #8877 anyhow :/

Reply:
Sorry, before https://github.com/kubernetes/test-infra/pull/8579